buttercup/cogs/find.py

```diff
@@ -68,84 +68,75 @@ def get_clean_transcription(data: Dict) -> str:
     return "---".join(parts[1:-1]).strip()
 
 
-class Find(Cog):
-    def __init__(self, bot: ButtercupBot, blossom_api: BlossomAPI) -> None:
-        """Initialize the Find cog."""
-        self.bot = bot
-        self.blossom_api = blossom_api
-
-    def to_embed(self, data: Dict) -> Embed:  # noqa: C901
-        """Convert the submission to a Discord embed."""
-        color = Color.from_rgb(255, 176, 0)  # Orange
-        status: str = "Unclaimed"
-
-        if "transcription" in data:
-            tr_link = f"[Link]({data['transcription']['url']})"
+def to_embed(data: Dict) -> Embed:  # noqa: C901
+    """Convert the submission to a Discord embed."""
+    color = Color.from_rgb(255, 176, 0)  # Orange
```

Comment:
I must say that I find this method a bit long and convoluted, since it is doing a lot, from extracting data from a dictionary to picking colors specific to Discord. Would we be able to split these two responsibilities?